packages/client/src/utils/generateInFolder.ts

```diff
@@ -50,9 +52,11 @@ export async function generateInFolder({
     previewFeatures,
   })
 
-  const outputDir = transpile
-    ? path.join(projectDir, 'node_modules/@prisma/client')
-    : path.join(projectDir, '@prisma/client')
+  const setOutputDir = relativeOutputDir ? path.join(projectDir, relativeOutputDir) : undefined
+
+  const outputDir =
+    setOutputDir ??
+    (transpile ? path.join(projectDir, 'node_modules/@prisma/client') : path.join(projectDir, '@prisma/client'))
 
   // if (transpile && config.generators[0]?.output) {
   //   outputDir = path.join(path.dirname(schemaPath), config.generators[0]?.output)
```

Comment:
Making the `output` generator option in `schema.prisma` work in tests instead would've been optimal, but that would require some investigation of why this was commented out in the first place, and being able to override that sounds useful, so let's settle on the solution with `relativeOutputDir` for now.